data/sitewide_alert.yml

```diff
@@ -0,0 +1,20 @@
+# Site Wide Alert setting that is displayed at the top of the website
+# https://designsystem.digital.gov/components/site-alert/
+#
+# display_alert: hide/show the sitewide alert on the site
+# is_emergency: true will display with orange emergency colors else it will display as blue info box
+# has_icon: hide/show icon
+# is_slim: sets a thinner banner height
+# header: bold text that appears next the icon at top of alert box
+# alerts: text field that supports markdown, will display a bulleted list if multiple, otherwise displays as standard text block
+
+data:
+  display_alert: false
```

Comment:
To be clear, we're going to hide the alert now?

Reply:
No, I've enabled it and updated the yaml to be consistent with the current live site messaging.